### PR TITLE
Convert slicer to handle single pages with multiple articles

### PR DESCRIPTION
**Pull request by seanmtracey**

### CODE REVIEW

File: slicer/index.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+
+const AWS = require('aws-sdk');
+const S3 = new AWS.S3();
+
+let currentJob;
+let isProcessing = false;
+
+function getJobFromSliceQueue(){
+	// Get item from queue and return it
+	return database.query('SELECT * FROM slice ORDER BY rand() LIMIT 1')
```
Comment (Lily2point0) [inline slicer/index.js]: select an item with status available

File: slicer/index.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+				.then(function(){
+					return job;
+				})
+		})
+	;
+}
+
+function deleteJobFromSliceQueue(id){
+	// Delete job from slice queue when it's been completed
+	return database.query(`DELETE * FROM slice WHERE id = ${id}`);
```
Comment (Lily2point0) [inline slicer/index.js:45]: check the status value before deleting

File: slicer/index.js [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+}
+
+function resetJobInSliceQueue(job){
```
Comment (Lily2point0) [inline slicer/index.js:48]: add retry count column in table

File: slicer/index.js [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+
+
+		});
+
+		Promise.all(articlesToProcess)
+			.then(function(){
+				isProcessing = false;
+			})
+			.catch(function(err){
+				debug('An error occurred proccessing one of the articles in the page', err);
```
Comment (Lily2point0) [inline slicer/index.js:156]: reset job here?

File: slicer/lib/image-processing.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+/*const gmPaths = [
+	`/tmp/resources/imagemagick/lib`,
+	`/tmp/resources/imagemagick/bin`,
+];
+
+console.log("PATH:", process.env['PATH']);
+process.env['PATH'] += `:${gmPaths.join(':')}`;
+console.log("ADJ PATH:", process.env['PATH']);*/
+
+var Promise = require('bluebird');
```
Comment (Lily2point0) [inline slicer/lib/image-processing.js]: `var`s to `let`s

File: slicer/index.js (hunk 2 above)
Comment (seanmtracey) [reply slicer/index.js:45]: The job is only deleted when it's been successfully completed. Would the check for the status be a little redundant?

File: slicer/index.js (hunk 3 above)
Comment (seanmtracey) [reply slicer/index.js:48]: 👍

**Review by Lily2point0 — APPROVED** [review]

**Pull request merged**